web/config/routes.ts

```diff
@@ -20,7 +20,7 @@ const routes = [
     component: './Metrics',
   },
   {
-    path: '/metrics',
+    path: '/dashboard',
     component: './Metrics',
   },
   {
```

Comment:
Hi @juzhiyuan 

Should I rename the `Metrics` component to `Dashboard`?

Reply:
Would better

Reply:
OK, I will improve it :)